pkg/registry/registry.go

```diff
@@ -22,6 +22,8 @@ import (
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 )
 
+const globalTestImage = "okteto.global/test"
```

Comment:
since it is only used once, I don't think it is helpful to have it as a constant, IMO it is more descriptive to use its value in this case when call `or.imageCtrl.ExpandOktetoGlobalRegistry("okteto.global/test")`

Reply:
The problem with that is that you don't know what is `okteto.global/test`. Having it as a constant makes you read that it's the test image for global registry, otherwise people could ask why that image and not any other